cva6/tests/custom/sv32/tests/satp_asid.S

```diff
@@ -0,0 +1,45 @@
+#include "../macros.h"
+.include "./trap_handler.S"
+.section .text
+.global _start
+
+_start:
+
+###########################################
+#    | M-mode Configurations and code |   #
+###########################################
+
+    la t2, trap_handler
+    WRITE_CSR (mtvec, t2)
+
+    li t0, 0x1FF
+    slli t0, t0, 22
+    WRITE_CSR(satp, t0)
+    READ_CSR(satp, t0)
+    li t1, 1
+    j exit
```

Comment:
Add a check to make sure that ASID length is same as we expected.

Reply:
Check has been added